_docs/tasks/ingress.md

```diff
@@ -153,53 +167,58 @@ to configure ingress behavior.
    
 1. Determine the secure ingress URL:
  
-    If your cluster is running in an environment that supports external load balancers,
-    use the ingress' external address:
+   * If your cluster is running in an environment that supports external load balancers,
```

Comment:
There's probably a better way to document this, but I think the whole procedure may change in the future, so IMO it's good enough as is for now.